include/mbedtls/check_config.h

```diff
@@ -1083,6 +1083,11 @@
 #error  "MBEDTLS_PKCS7_C is defined, but not all prerequisites"
 #endif
 
```

Comment:
@daverodgman Why are we still adding new crypto-using features without support for PSA? We could make PKCS7 PSA-only from the start, like we did for TLS 1.3. This would avoid having to migrate it later.